<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnaryOperationsWidget</class>
 <widget class="QWidget" name="UnaryOperationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="invert_checkbox">
        <property name="text">
         <string>Invert</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="binarize_checkbox">
        <property name="text">
         <string>Binarize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dct_checkbox">
        <property name="text">
         <string>DCT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="idct_checkbox">
        <property name="text">
         <string>iDCT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QPushButton" name="perform_button">
     <property name="text">
      <string>Perform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
